app/src/main/java/org/fossasia/openevent/general/di/Modules.kt

```diff
@@ -84,4 +87,21 @@ val networkModule = applicationContext {
                 .build()
     }
 
+}
+
+val databaseModule = applicationContext {
+
+    bean {
+        var INSTANCE: OpenEventDatabase? = null
+
+        if (INSTANCE == null) {
+            synchronized(OpenEventDatabase::class.java) {
+                if (INSTANCE == null) {
+                    INSTANCE = Room.databaseBuilder(androidApplication(),
+                            OpenEventDatabase::class.java, "open_event_database")
+                            .build()
+                }
+            }
+        }
```

Comment:
This will cause dry run test to fail. This should be lazily evaluated. Update dryrun test and try to run the test